phys/module_physics_init.F

```diff
@@ -2993,14 +3134,19 @@ SUBROUTINE bl_init(STEPBL,BLDT,DT,RUBLTEN,RVBLTEN,RTHBLTEN,  &
                      its,ite, jts,jte, kts,kte                 )
 
 
-!URBAN
-          IF ((SF_URBAN_PHYSICS.eq.1).OR.(SF_URBAN_PHYSICS.EQ.2).OR.(SF_URBAN_PHYSICS.EQ.3)) THEN
+!URBAN. Add TEB with option 4.
+          IF ((SF_URBAN_PHYSICS.eq.1).OR.(SF_URBAN_PHYSICS.EQ.2).OR.(SF_URBAN_PHYSICS.EQ.3).OR.(SF_URBAN_PHYSICS.EQ.4)) THEN
```

Comment:
@dmey 
Even though you are following the existing template, it would be better if these tests were NOAHUCMSCHEME, BEPSCHEME, BEP_BEMSCHEME (instead of 1, 2, 3).
If the resulting line is > 130 characters, break it in to two lines.

Reply:
Do you want me to change to use names in this PR? There are other instances where this happen so wanted to keep consistency with the current code. I can send you a separate (non-TEB) related PR where we just replace all of that if you want.

Reply:
@dmey
Let us leave the name vs number change until AFTER this PR is complete. Then come back and issue a clean-up PR (possibly for your code and for similar changes from other developers).